sql/core/src/test/resources/sql-tests/results/number-format.sql.out

```diff
@@ -19,16 +19,24 @@ struct<2147483648:bigint,(-2147483649):bigint>
 
 
 -- !query 2
-select 9223372036854775808, -9223372036854775809
+select 9223372036854775807, -9223372036854775808
```

Comment:
Can I add it in a separate pull request? I want to add all literal parsing here, but don't want to distract this pull request.